src/plot.jl

```diff
@@ -0,0 +1,156 @@
+export plot_single_flow, plot_graph, plot_assets_capacity
+
+"""
+    plot_single_flow(graph::MetaGraph,              asset_from::String, asset_to::String, rp::Int64,)
+    plot_single_flow(energy_problem::EnergyProblem, asset_from::String, asset_to::String, rp::Int64,)
+
+Plot a single flow over a single representative period,
+given a graph or energy problem, the "from" (exporting) asset, the "to" (importing) asset, and the representative period.
```

Comment:
```suggestion
Plot a single flow over a single representative period, given a graph or energy problem,
the "from" (exporting) asset, the "to" (importing) asset, and the representative period.
```
or somethings closes to 100, because I haven't counted the columns.